tests/drivers/spi/spi_loopback/testcase.yaml

```diff
@@ -24,4 +24,4 @@ tests:
     extra_args: OVERLAY_CONFIG="overlay-stm32-spi-dma.conf"
     filter: CONFIG_SOC_FAMILY_STM32
     platform_allow: nucleo_g474re nucleo_f207zg nucleo_f429zi nucleo_f746zg nucleo_wb55rg
-        nucleo_wl55jc nucleo_h743zi
+        nucleo_l152re nucleo_wl55jc nucleo_h743zi
```

Comment:
don't you need to add stm32f3_disco to platform_allow ?

Reply:
The choice here is to run this test on stm32f3_disco without DMA